dts/bindings/memory-controllers/st,stm32-bbram.yaml

```diff
@@ -10,6 +10,8 @@ description: |
   by system reset or when the device wakes up from Standby mode. They are
   reset by a backup domain reset.
 
+  NOTE: Since STM32 BBRAM is in RTC backup domain, STM32 RTC node must be enabled if STM32 BBRAM node is enabled.
```

Comment:
Could you split in 2 lines. 80char/line max is recommended in YAML bindings files.